### ISSUE TITLE
aws_wafregional_geo_match_set cannot be added to a rule

### ISSUE THREAD

**Issue by dcohan** [terraform-providers/terraform-provider-aws#4039]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
0.11.5

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_wafregional_geo_match_set
- aws_wafregional_rule
- aws_wafregional_rule_group (probably)

### Terraform Configuration Files
```hcl
resource "aws_wafregional_geo_match_set" "geo_match_set" {
  name = "geo_match_set"

  geo_match_constraint {
    type  = "Country"
    value = "AR"
  }
}

resource "aws_wafregional_rule" "georule" {
  depends_on  = ["aws_wafregional_geo_match_set.geo_match_set"]
  name        = "${var.group}-${var.environment}-waf-geo-rule"
  metric_name = "${var.group}${var.environment}wafgeorule"

  predicate {
    data_id = "${aws_wafregional_geo_match_set.geo_match_set.id}"
    negated = false
    type    = "GeoMatch"
  }
}
```

### Expected Behavior
Created a match set and a rule using that geo match set

### Actual Behavior
The resource aws_wafregional_rule don't expect the type GeoMatch

**Comment by bflad** [comment]
PR submitted: #4046

**Comment by bflad** [comment]
The fix for the validation to allow all predicate types in the `aws_wafregional_rule` has been merged into master and will release with v1.14.0 of the AWS provider, likely by the end of this week.

**Comment by bflad** [comment]
This has been released in [version 1.14.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1140-april-06-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
